src/Microsoft.Azure.SignalR.Common/ClientInvocation/IRoutedClientResultsManager.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.SignalR.Protocol;
+
+namespace Microsoft.Azure.SignalR
+{
+    internal interface IRoutedClientResultsManager
+    {
+        void AddInvocation(string connectionId, string invocationId, string callerServerId, string instanceId, CancellationToken cancellationToken);
+
+        bool TryCompleteResult(string connectionId, CompletionMessage message);
+
+        bool TryGetInvocationReturnType(string invocationId, out Type type);
+
+        bool ContainsInvocation(string invocationId);
```

Comment:
Yes. 
But considering when we just want to know whether a `invocationId` exists, using `TryGetInvocationReturnType` is some kind of confusing. 
Because `...ReturnType` has nothing to do with our logic.
This method is indeed unnecessary but I'm not sure if it should be removed.